### PR TITLE
Issues #SB-00 feat: adding for location master data api's

### PR DESCRIPTION
**Pull request by manzarul**
- [ ] Have you followed [Contribution Guidelines](http://www.sunbird.org/contributions/contribution_guidelines/)
  Adding new api's for location master data service.
-----

### CODE REVIEW

**Comment by manzarul** [comment]
Location master data service : list of states, district,blocks, clusters.

**Comment by lakhanmandloi** [comment]
Hello @manzarul,<br/><br/> _This is a auto-generated response._ <br/> Click the URL for review pull request - http://sunbird-docs-qa.s3-website.ap-south-1.amazonaws.com/pr/389. <br/><br/> **Modified pages are -** <br />- http://sunbird-docs-qa.s3-website.ap-south-1.amazonaws.com/pr/389/apis/locationMasterDataService/ <br /> - http://sunbird-docs-qa.s3-website.ap-south-1.amazonaws.com/pr/389/apis/locationMasterDataService/ <br /> - http://sunbird-docs-qa.s3-website.ap-south-1.amazonaws.com/pr/389/apis/locationMasterDataService/ <br /> - http://sunbird-docs-qa.s3-website.ap-south-1.amazonaws.com/pr/389/apis/locationMasterDataService/ <br />  <br/> **Modified sections are -** <br />

**Comment by Sunitashelar** [comment]
@manzarul The 'org API modification for SK' - is this supposed to be a new file or is it an amendment to the Org API? If its a new file, the use of the name SK is making it too specific. 
@kochhar after a discussion with Anand, it appears that these services and APIs (4) are specifically created to upload data into the registry from Shalakosh for DIKSHA. Please let us know if we can add such specific files to Sunbird API reference repo. Also, if they should be added, I am not OK with using the name used for the API mentioned above Please suggest a name change.
@nawabahmadreshi please follow up. PR will be merged and content published only after we get clarity on the above.

File: apis/locationMasterDataService/lmds.yaml [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+    provided in CSV format.
+  license:
+    name: GNU Affero General Public License v3.0
+    url: 'https://github.com/project-sunbird/sunbird-commons/blob/master/LICENSE'
+host: staging.open-sunbird.org
+basePath: /api/shaalakosh/v1
+schemes:
+  - https
+paths:
+  /location/state/upload:
```
Comment (nawabahmadreshi) [inline apis/locationMasterDataService/lmds.yaml]: @manzarul 
The URIs are not appropriate as it does not fall in line with Sunbird terms .

The URIs are more specific to DIKSHA use case, but on Sunbird we should be using generic naming conventions

@bvinayakumar

**Comment by nawabahmadreshi** [comment]
Accepted Bulk Upload and Location APIs 
Requested changes to Data Service APIs

**Comment by nawabahmadreshi** [comment]
done except data service which is diksha related

**Pull request closed without merging**